lib/matplotlib/tests/test_collections.py

```diff
@@ -646,12 +646,12 @@ def test_lslw_bcast():
     col.set_linestyles(['-', '-'])
     col.set_linewidths([1, 2, 3])
 
-    assert col.get_linestyles() == [(None, None)] * 6
-    assert col.get_linewidths() == [1, 2, 3] * 2
+    assert_equal(col.get_linestyles(), [(None, None)] * 6)
```

Comment:
because `col.get_linewidths` can now return an array rather than a list, due to the switch from
`self._us_lw = self._get_value(lw)`
to
`self._us_lw = np.atleast_1d(np.asarray(lw))`